src/test/kotlin/org/rust/ide/formatter/RustFormatterTestCase.kt

```diff
@@ -52,6 +52,11 @@ class RustFormatterTestCase : FormatterTestCase() {
         doTest()
     }
 
+    fun testAlignTypeParamsOn() {
+        custom().ALIGN_TYPE_PARAMS = true
+        doTest()
+    }
```

Comment:
Again, not related to PR, but a more DSLy API would be

```Kotlin
   fun testAlignTypeParamsOn() = custom {
       ALIGN_TYPE_PARAMS = true
   }
```

where `custom` is defined as

```Kotlin
fun custom(block: RustCustomSettings.() -> Unit) {
    val settings = settings.getCustomSettings(RustCodeStyleSettings::class.java)
    settings.block() // <- this is pure awesomeness, because the extension function here is a parameter. 
    doTest()
}
```